dlc_developer_config.toml

```diff
@@ -6,7 +6,7 @@ partner_developer = ""
 ei_mode = false
 # Please only set it to true if you are preparing a NEURON related PR
 # Do remember to revert it back to false before merging any PR (including NEURON dedicated PR)
-neuron_mode = false
+neuron_mode = true
```

Comment:
You will need to set `neuron_mode` to `true` in order to start CI builds for Neuron DLCs.